admin/includes/init_includes/init_admin_auth.php

```diff
@@ -66,7 +68,7 @@
     }
   }
 
-  if (STORE_NAME == '' || STORE_OWNER == '') {
+  if ((STORE_NAME == '' || STORE_OWNER == '')) {
```

Comment:
poop. Had started to make changes here to test $authError then realized hey were not needed. have reverted the parentheses